scripts/generate-renode-scripts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import csv
+import zlib
+import argparse
+
+def generate_ethmac(data):
```

Comment:
Please add docstring and [maybe type hints](https://docs.python.org/3/library/typing.html).

Reply:
I added docstrings to all functions.